scripts/core/editor3/reducers/find-replace.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable no-unused-vars */ // eslint complains about EditorChangeType not being used
```

Comment:
You can use `eslint-disable-next-line` to avoid having to re-enable rules.

```
// eslint-disable-next-line no-unused-vars
import {EditorChangeType} from 'draft-js';

import {SelectionState, Modifier, EditorState} from 'draft-js';
```